lib/content-services/src/lib/search/services/search-configuration.service.ts

```diff
@@ -32,8 +31,8 @@ export class SearchConfigurationService implements SearchConfigurationInterface
      * @param skipCount The offset of the start of the page within the results list
      * @returns Query body defined by the parameters
      */
-    generateQueryBody(searchTerm: string, maxResults: number, skipCount: number): QueryBody {
-        const defaultQueryBody: QueryBody = {
+    generateQueryBody(searchTerm: string, maxResults: number, skipCount: number): SearchRequest {
```

Comment:
Could you please update search-configuration.service.md file with new types?

Reply:
updated docs, the public api stil remains the same for now